services/user/Tokens/include/services/user/tokenTypes.hpp

```diff
@@ -12,26 +12,34 @@ namespace UserService
 
    struct Precision
    {
-      static constexpr uint8_t          precisionMin = 0;
-      static constexpr uint8_t          precisionMax = 16;
-      static constexpr std::string_view errorInvalid = "Value exceeds allowed Precision range";
+      static constexpr uint8_t precisionMin = 0;
+      static constexpr uint8_t precisionMax = 8;
```

Comment:
I don't understand the reasoning for choosing a particular value of `precisionMax`. Is there some way that 16 causes a problem?

Reply:
It's just for ergonomics. Can be increased later if there's actually a need for >8 decimals of precision.